### ISSUE TITLE
fs.dir with embed resources

### ISSUE THREAD

**Issue by filt3rek** [HeapsIO/heaps#1090]
Hej,

I'm new to heaps so sorry if maybe it's a stupid question but I wonder why `dir` isn't available in https://github.com/HeapsIO/heaps/blob/master/hxd/fs/EmbedFileSystem.hx#L246 ?

For my needs I can fill the body of this function like that : 
```haxe
public function dir( path : String ) : Array<FileEntry> {
	//throw "Not Supported";
	#if !macro
	return [ for( o in getRoot().get( path ).iterator() ) o ];
	#else
	return  null;
	#end
}
```

**Comment by rcstuber** [comment]
EmbedFilesystem has the 'subFiles' function to get a list of entries. Not sure why they have different APIs though.

**Comment by filt3rek** [comment]
Thanks for your answer !
Ok, so maybe we could kind of unify all that so it would have the same way to work ?

**Comment by trethaller** [comment]
Sorry I fixed this a few days later with 4050d6869029cbc514863f2af0bb64111f5a5c56 but didn't reference the issue
